python/private/bzlmod/pip.bzl

```diff
@@ -217,6 +222,32 @@ def _pip_impl(module_ctx):
     # Build all of the wheel modifications if the tag class is called.
     _whl_mods_impl(module_ctx)
 
+    _overriden_whl_set = {}
+    whl_overrides = {}
```

Comment:
Good catch, gazelle also supports overrides for cases when `module_ctx.is_isolated == True`, but I am not sure how that works in practice and what it would mean in our case. https://github.com/bazelbuild/bazel-gazelle/blob/545aa35aea1753dff405590c02080fcc0ff0c549/internal/bzlmod/go_deps.bzl#L46